ffcx/codegeneration/C/integrals.py

```diff
@@ -69,14 +69,21 @@ def generator(ir: IntegralIR, options):
     else:
         code["tabulate_tensor_complex64"] = ".tabulate_tensor_complex64 = NULL,"
         code["tabulate_tensor_complex128"] = ".tabulate_tensor_complex128 = NULL,"
+    if options.get("cuda"):
+        code["tabulate_tensor_cuda"] = (
+            f".tabulate_tensor_cuda = tabulate_tensor_cuda_{factory_name}"
```

Comment:
Propose `tabulate_tensor_cuda_nvrtc` to differentiate with `tabulate_tensor_cuda` or ``tabulate_tensor_cuda_nvcc` in the future?